src/lib/rpc/bestRpcTracker.ts

```diff
@@ -79,7 +83,7 @@ function trackRpcProviders({ warmUp = false } = {}) {
       const hasMultipleProviders = Object.keys(providers).length > 1;
       const isUnusedChain =
         !lastUsage || differenceInMilliseconds(Date.now(), lastUsage) > DISABLE_UNUSED_TRACKING_TIMEOUT;
-      const isChainTrackingEnabled = (warmUp || !isUnusedChain) && hasMultipleProviders;
+      const isChainTrackingEnabled = !isLargeAccount && (warmUp || !isUnusedChain) && hasMultipleProviders;
```

Comment:
I think this part of requirements wasn't implemented:

>If paid RPC does not work, usual logic of RPC selection must be used (competitive selection, switching, etc, including free RPCs)

Reply:
My initial thought was that after the last ABs, we proved Alchemy is reliable and don't need this 'fallback for fallback.' However, after syncing with G, it became clear that it's better to do it.
Thank you for highlighting this point.